backend/apps/owasp/api/search/chapter.py

```
@@ -6,7 +6,18 @@
 
 
 def get_chapters(query, attributes=None, limit=25, page=1, searchable_attributes=None):
-    """Return chapters relevant to a search query."""
+    """Return chapters relevant to a search query.
+
+    Args:
+        query (str): The search query string.
+        attributes (list, optional): List of attributes to retrieve. Defaults to None.
+        limit (int, optional): Number of results per page. Defaults to 25.
+        page (int, optional): Page number for pagination. Defaults to 1.
+        searchable_attributes (list, optional): Attributes to restrict the search to. Defaults to None.
```

Ruff (0.8.2)

Line too long (103 > 99)

(E501)